ScreenToGif/ImageUtil/Gif/LegacyEncoder/PixelUtilOld.cs

```diff
@@ -16,6 +16,8 @@ public class PixelUtilOld
         private readonly Bitmap _source = null;
         private BitmapData _bitmapData = null;
 
+        readonly object lockObject = new object();
```

Comment:
private readonly object _lockObject = new object();

Underlines for private variables.